javascript/javascript3/week1/giphyApp/giphy.js

```diff
@@ -0,0 +1,34 @@
+const btnTag = document.getElementById("submit");
+
+const url =
+  "https://api.giphy.com/v1/gifs/search?api_key=mSA30PH3U1iP124OtC2BNkS9NJyiTfRC";
+
+btnTag.addEventListener("click", async (e) => {
+  e.preventDefault();
+  const searchTerm = document.getElementById("search-id").value;
+  const resultNumber = document.getElementById("count").value || 1;
+  searchGiphy(searchTerm, resultNumber);
+});
+const searchGiphy = async (term, count) => {
+  try {
+    const apiKey = "mSA30PH3U1iP124OtC2BNkS9NJyiTfRC";
```

Comment:
API key in plain text

Reply:
will keep api key in secret env.json file